docs/dub-reference/subpackages.md

```diff
@@ -46,6 +46,12 @@ For example, with a root package named `package` with a `mylibrary` sub-package,
 a `Have_package_mylibrary` version will be automatically defined by DUB,
 and will be visible for when the sub-package is used as dependency.
 
+Specified build types and configurations (e.g., via the command-line
+interface) are applied to the sub-package, and are taken from the sub-package recipe.
+For example, executing `dub build :mylibrary -c experimental`, the `experimental`
+configuration must exist in the sub-package recipe.
+If a build type exists in both packages, both will be applied.
```

Comment:
since you mention the recipe as well, can you add an example for `subConfiguration ":mylibrary" "experimental"` as well?